File: Database/Assembly_SRPG/WeaponParam.cs
namespace SRPG
{
    using System;

    public class WeaponParam
    {
        public string iname;
        public OInt atk;
        public OInt formula;

        public WeaponParam()
        {
            base..ctor();
            return;
        }

        public bool Deserialize(JSON_WeaponParam json)
        {
            if (json != null)
            {
                goto Label_0008;
            }
            return 0;
        Label_0008:
            this.iname = json.iname;
            this.atk = json.atk;
            this.formula = json.formula;
            return 1;
        }

        public WeaponFormulaTypes FormulaType
        {
            get
            {
                return this.formula;
            }
        }
    }
}

File: Database_V2/ParamFunctions/WeaponParam.py
def WeaponParam(json):
    this={}#WeaponParamjson)
    if 'iname' in json:
        this['iname'] = json['iname']
    if 'atk' in json:
        this['atk'] = json['atk']
    if 'formula' in json:
        this['formula'] = json['formula']
    return this
